manifest.versions.json

```
@@ -215,13 +215,13 @@
     "powershell|10.0|Linux|x64|sha": "ac4eb5cb83e130aa7cc9c856245635d3f4e624b8d4e12933f5cd534f02545e2f4c61b7f63bc899d7ad797a70ec5c2d7bca33e57ea6eb7da0a410808d3ad2cf94",
     "powershell|10.0|Windows|x64|sha": "6f4518a04d9fd3ade4c7fa76574b29f6abb7e8a37793fd2c14a123700dfd8fa69ff4a200287ac909942bd687209ead999f53fbb2727ae96200d669564e1935eb",
 
-    "yarp|2.3|build-version": "2.3.0-preview.1.25060.1",
-    "yarp|2.3|product-version": "2.3.0-preview.1",
+    "yarp|2.3|build-version": "3.0.0-preview.1.25171.2",
+    "yarp|2.3|product-version": "2.3.0-preview.2",
```

Looks like this version should be updated:

```suggestion
    "yarp|3.0|build-version": "3.0.0-preview.1.25171.2",
    "yarp|3.0|product-version": "3.0.0-preview.1",
```

Also applies to all other `yarp|*` variables below.

---

The YARP binaries are still from the 2.3.0 YARP release, it's just the daily build definition of yarp automatically naming the zip file 3.0.

---

Is this a temporary situation? I don't understand which version of YARP we are delivering previews for. Is the next version a 2.3.0 stable release, 2.3.1, 2.4, or 3.0? The versions here should reflect that.